### DISCUSSION TITLE
Some bug with PHP 8.1.16 - Deprecated: Optional parameter $conta declared before required parameter $psc is implicitly treated as a required parameter in D:\www\eclipse-workspace-php\Falesia\amb\ser.php on line 259

### DISCUSSION THREAD

**Discussion by Fede72** [giuliopons/adadmin#105] [General]
I got his error with PHP 8.1.16:

Deprecated: Optional parameter $conta declared before required parameter $psc is implicitly treated as a required parameter in D:\www\eclipse-workspace-php\Falesia\amb\ser.php on line 259

![bug1](https://github.com/giuliopons/adadmin/assets/10374924/ca9bb54a-eafb-4140-8398-2cd2cb0fc026)

**Comment by giuliopons** [comment]
Thank you
I will upload the new version this weekend
Giulio
